tgui/packages/tgui/interfaces/PortableTurret.js

```diff
@@ -0,0 +1,102 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Button, LabeledList, NoticeBox, Section } from '../components';
+import { Window } from '../layouts';
+
+export const PortableTurret = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    locked,
+    on,
+    lethal,
+    lethal_is_configurable,
+    targetting_is_configurable,
+    check_weapons,
+    neutralize_noaccess,
+    neutralize_norecord,
+    neutralize_criminals,
+    neutralize_all,
+    neutralize_unidentified,
+    neutralize_cyborgs,
+  } = data;
+  return (
+    <Window
+      width={300}
+      height={300}>
+      <Window.Content>
+        <NoticeBox>
+          Swipe an ID card to {locked ? 'unlock' : 'lock'} this interface.
+        </NoticeBox>
+        <Fragment>
```

Comment:
Unsure if this needs to be inside a `Fragment`

Reply:
Removed